iron-swipeable-container.html

```diff
@@ -0,0 +1,266 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`<iron-swipeable-container>` is a container that allows any of its nested
+children (native or custom elements) to be swiped away. By default it supports
+a curved or horizontal transition, but the transition duration and properties
+can be customized.
+
+Example:
+
+    <iron-swipeable-container>
+      <div>I can be swiped</div>
+      <paper-card heading="Me too!"></paper-card>
+    </iron-swipeable-container>
+
+To disable swiping on individual children, you must give them the `.disable-swipe`
+class. Alternatively, to disable swiping on the whole container, you can use its
+`disable-swipe` attribute:
+
+    <iron-swipeable-container>
+      <div class="disable-swipe">I cannot be swiped be swiped</div>
+      <paper-card heading="But I can!"></paper-card>
+    </iron-swipeable-container>
+
+    <iron-swipeable-container disable-swipe>
+      <div>I cannot be swiped</div>
+      <paper-card heading="Me neither :("></paper-card>
+    </iron-swipeable-container>
+
+It is a good idea to disable text selection on any of the children that you
+want to be swiped:
+
+    .swipe {
+      -moz-user-select: none;
+      -ms-user-select: none;
+      -webkit-user-select: none;
+      user-select: none;
+      cursor: default;
+    }
+
+@group Iron Elements
+@element iron-swipeable-container
+@demo demo/index.html
+-->
+
+<dom-module id="iron-swipeable-container">
+  <template>
+    <content id="content"></content>
+  </template>
+</dom-module>
+
+<script>
+
+  Polymer({
+    is: 'iron-swipeable-container',
+
+    /**
+     * Fired when a child element is swiped away.
+     *
+     * @event iron-swipe-away
+     */
+
+    properties: {
+      /**
+       * The style in which to swipe the card. Currently supported
+       * options are `curve | horizontal`. If left unspecified, the default
+       * is assumed to be `curve`.
```

Comment:
I double checked with the MD motion interaction designer and he suggested the horizontal to be the default, especially since iOS is the only MD thing that uses the curve animation (we'll keep it, but just flip the boolean)